android/jni/app/organicmaps/Framework.cpp

```diff
@@ -1106,7 +1106,7 @@ Java_app_organicmaps_Framework_nativeGetMovableFilesExts(JNIEnv * env, jclass)
 JNIEXPORT jobjectArray JNICALL
 Java_app_organicmaps_Framework_nativeGetBookmarksFilesExts(JNIEnv * env, jclass)
 {
-  const vector<string> exts = { kKmzExtension, kKmlExtension, kKmbExtension, kGpxExtension };
+  std::array<std::string_view, 4> exts = { kKmzExtension, kKmlExtension, kKmbExtension, kGpxExtension };
```

Comment:
I'm not insisting, I assumed that it can be easy, but looks like the reality is worse )

Anyway, if we are already here, it makes sense to fix it properly. I hope that @cyber-toad learned something new in the process )

Reply:
Yes, some issue in jni bridge for Android, I'm looking into it.